package.json

```diff
@@ -1,6 +1,4 @@
 {
-  "name": "coala-bears",
```

Comment:
Updated. Now using `OrderedDict` to maintain the order instead of using the attribute `sort_keys=True`.